app/src/main/java/com/bitchat/android/onboarding/PermissionExplanationScreen.kt

```diff
@@ -94,10 +96,10 @@ fun PermissionExplanationScreen(
                 }
                 
                 Text(
-                    text = "• bitchat doesn't track you or collect personal data\n" +
```

Comment:
This is the wrong way to do translations because those splits might not make sense in other languages.